src/lib/Scenes/Onboarding/OnboardingCreateAccount/__tests__/OnboardingCreateAccount-tests.tsx

```diff
@@ -0,0 +1,148 @@
+import { Checkbox } from "lib/Components/Bidding/Components/Checkbox"
+import { Input } from "lib/Components/Input/Input"
+import { renderWithWrappers } from "lib/tests/renderWithWrappers"
+import React from "react"
+import { __globalStoreTestUtils__ } from "../../../../store/GlobalStore"
+import { flushPromiseQueue } from "../../../../tests/flushPromiseQueue"
+import { OnboardingCreateAccount } from "../OnboardingCreateAccount"
+
+const goBackMock = jest.fn()
+const replaceMock = jest.fn()
+
+const navigationMock = {
+  goBack: goBackMock,
+  replace: replaceMock,
+}
+
+const mockFetch = jest.fn()
+
+;(global as any).fetch = mockFetch
+
+function mockFetchResponseOnce(response: Partial<Response>) {
+  mockFetch.mockResolvedValueOnce(response)
+}
+function mockFetchJsonOnce(json: object, status: number = 200) {
+  mockFetch.mockResolvedValueOnce({
+    status,
+    json: () => Promise.resolve(json),
+  })
+}
+
+describe("OnboardingCreateAccount", () => {
+  beforeEach(() => {
+    mockFetch.mockClear()
+  })
+
+  it("form validation works properly", async () => {
+    const tree = renderWithWrappers(<OnboardingCreateAccount navigation={navigationMock as any} route={null as any} />)
+
+    const signUpButton = tree.root.findAllByProps({ testID: "signUpButton" })[0]
```

Comment:
nonblocking:
```suggestion
    const signUpButton = tree.root.findByProps({ testID: "signUpButton" })
```
i prefer this pattern since you get better error messages if the button is not present for some reason, or if there's multiple instances of the same button

Reply:
yes, that makes sense actually. I will change it